fhir-path/src/main/java/com/ibm/fhir/path/function/ToStringFunction.java

```diff
@@ -38,7 +39,10 @@ public int getMaxArity() {
 
     @Override
     public Collection<FHIRPathNode> apply(EvaluationContext evaluationContext, Collection<FHIRPathNode> context, List<Collection<FHIRPathNode>> arguments) {
-        checkSingleton(context);
+        if(!isSingleton(context)) {
```

Comment:
I think this one should return empty if the input collection is empty, but I don't think it does with this